test/gpdb/run_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+DOCKERIMAGE="$1"
+LOG_DIR="$2"
+SCRIPT_PATH=$(dirname $(realpath -s $0))
+PROJECT_ROOT=$(dirname $(dirname "$SCRIPT_PATH"))
+
+if [ -z "$(docker images -q pgbackrest:test 2> /dev/null)" ]; then
+  docker build -t pgbackrest:test -f $SCRIPT_PATH/Dockerfile \
+  $PROJECT_ROOT --build-arg GPDB_IMAGE=$DOCKERIMAGE
+fi
```

Comment:
>maybe it's better to always run the script from the root folder of the project?

Do you propose to move run_docker.sh to project root?

Reply:
> Do you propose to move run_docker.sh to project root?

No. I mean, if we run the script from the root folder, then we don't need to calculate `SCRIPT_PATH` and `PROJECT_ROOT`.
The script itself can stay where it is and you can run it as:
`bash test/gpdb/run_docker.sh`

Reply:
I dunno, I wouldn't pay attention to this aspect. If someone consider current variant inconvinient, I'll change. The whole quality of script is quite bad in my opinion, but I have very little idea how I can improve it. Maybe use separate framework like pg_regress, but there is no much sense, because testing pgbacckrest requires lots of shell commands and manual config setting.